api/types/samlsp/samlsp.go

```diff
@@ -20,6 +20,9 @@ const (
 	// GCPWorkforce is a SAML service provider preset name for Google Cloud Platform
 	// Workforce Identity Federation.
 	GCPWorkforce = "gcp-workforce"
+	// Unspecified preset type is used in the Web UI to denote a generic SAML service
+	// provider preset.
+	Unspecified = "unspecified"
```

Comment:
Very nitpicky but I like "generic" name instead of "unspecified" for this as you have in your comment. lgtm either way though.

Reply:
Sounds good, I will keep this as is this time as UI is already shipped with  `unspecified ` some releases ago :(